src/hooks/useVote.js

```diff
@@ -0,0 +1,192 @@
+import { useEffect, useState, useMemo } from 'react'
+import { connectorConfig } from '../current-environment'
+import { createAppHook } from '@aragon/connect-react'
+import { formatTokenAmount } from '@aragon/ui'
+import connectVoting from '@aragon/connect-disputable-voting'
+import { useWallet } from '../providers/Wallet'
+import { useMounted } from '../hooks/useMounted'
+import { ProposalNotFound } from '../errors'
+import { useOrgApps } from '../providers/OrgApps'
+
+const useDisputableVoting = createAppHook(
+  connectVoting,
+  connectorConfig.disputableVoting
+)
+
+export function useVote(proposalId) {
+  const [vote, loading] = useVoteSubscription(proposalId)
+  return [vote, loading]
+}
```

Comment:
The intention is to do common unit formatting here, `useVoteSubscription` should be providing the original values.